docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.9"
+
+services:
+  app:
+    build:
+      context: .
+    ports:
+      - "8000:8000"
+    env_file:
+      - .env
+
+  db:
+    env_file:
+      - .env
+
+  wait-for-db:
+    depends_on:
+      - db
+    command: ["wait-for-it", "--timeout=30", "--", "sh", "-c", "python manage.py migrate"]
```

Comment:
Ensure that the `wait-for-it` script is available in the container. Alternatively, consider using a more integrated solution like `wait-for` in the Dockerfile or entrypoint script.